PyLeap/Views/Project Views/ProjectCardView.swift

```diff
@@ -9,51 +9,99 @@ import SwiftUI
 
 struct ProjectCardView: View {
     
-    // Params
-    let fileTransferClient: FileTransferClient?
     var project: Project
+    //@AppStorage("onboarding") var onboardingSeen = false
+    
     // Data
     @Environment(\.presentationMode) var presentationMode
+    
     @StateObject var model = ProjectViewModel()
     @StateObject var downloadModel = DownloadViewModel()
     
-    @State private var fileContents = ""
+    @State private var filename = "/code.py"
+    @State private var consoleFile = "/boot_out.txt"
```

Comment:
What do you use this for? It'll only contain the output from boot.py (not code.py).